### ISSUE TITLE
Use `master` branch on Github for component usage report script

### ISSUE THREAD

**Issue by k80bowman** [department-of-veterans-affairs/vets-design-system-documentation#1152]

**Comment by caw310** [comment]
@Andrew565 @jamigibbs - can we close this?

**Comment by jamigibbs** [comment]
@caw310 This functionality does not currently exist for the component usage report and if we plan on continuing to use it (after all of the React components have been deprecated), then I think it would still be a nice update to the script.

**Comment by Andrew565** [comment]
Agreed, though we've since moved `master` to `main`, for what that's worth.
